lib/public/views/Runs/Details/RunDetailsModel.js

```diff
@@ -41,7 +41,8 @@ export class RunDetailsModel extends Observable {
         this._runId = null;
         this._runDetails = RemoteData.notAsked();
         this._eorReasonTypes = RemoteData.notAsked();
-        this._allDetectors = RemoteData.notAsked();
+        this._allDetectors$ = detectorsProvider.all$;
+        this._allDetectors$.bubbleTo(this);
```

Comment:
maybe also `this._detectors$` like in other classes, I mean up to you, I just don't like this name

Reply:
Renamed, but not the getter to be clear from outter context that we are not getting specific detector list